Header header

int32 id
geometry_msgs/Point room_length
geometry_msgs/Point room_center
